src/Parts.elm

```diff
@@ -74,7 +122,7 @@ type alias Update model msg =
 {-| Standard TEA view function type. 
 -}
 type alias View model a = 
-  model -> a
+  model -> Html a
```

Comment:
Shouldn't that be in the model? I am relatively new to elm, so an example would be helpful.